12/umbraco-cms/tutorials/creating-and-distributing-a-package.md

```diff
@@ -62,7 +62,7 @@ dotnet new umbraco-v8-package -n PackageWorkshop -d
 
 This will create a new package called `PackageWorkshop`, and add a custom Dashboard for us to use in this tutorial. By default you will also get a Github Action added that we will return to later.
 
-Afer running you will have a folder called `PackageWorkshop`, inside that you will have your site and solution files. So try to open it in Visual Studio or Rider by opening the `PackageWorkshop.sln` file.
+After running you will have a folder called `PackageWorkshop`, inside that you will have your site and solution files. So try to open it in Visual Studio or Rider by opening the `PackageWorkshop.sln` file.
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?